worldinfo.cpp

```diff
@@ -39,7 +39,11 @@ void WorldInfo::clear() {
 }
 
 
-bool WorldInfo::parseFolder(const QDir &path)
+// used to generate menu File->Open World
+// parse only basic stuff in the world folder:
+//  does it look like a Minecraft world
+//  does the world have a custom name
```

Comment:
This should have been a doxy-style comment in the header instead. Here it's buried.